src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -1238,84 +1257,84 @@ internal async Task Cleanup(bool async, bool connectionClosing = false, bool isD
     /// </summary>
     /// <param name="ordinal">The zero-based column ordinal.</param>
     /// <returns>The value of the specified column.</returns>
-    public override bool GetBoolean(int ordinal) => GetFieldValue<bool>(ordinal);
+    public override bool GetBoolean(int ordinal) => GetFieldValueCore<bool>(ordinal);
```

Comment:
This is just to avoid the virtual dispatch? No objections, but shouldn't GDV take care of it?

Reply:
GDV doesn't work at all for GVMs :/